### PR TITLE
added information about project in readme-filde

### PR DESCRIPTION
**Pull request by mudritskiy**
goin to add screenshots
need advices

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
+## About
+Quick access to repo search by main characteristics (basically name). Viewing brief information of the selected repo.
+
+## Screenshots
+
+## Features (What makes your project stand out?)
+Allows using a mobile phone to use the search for repo on github, in an easy and convenient way.
+* Having all the features of github search you can find needed repo on a walk.
+* No need to login on github or use browser.
+* Minimalistic design only for breif search. Leave details for later.
```
Comment (Granfalloner) [inline README.md]:
```suggestion
* Minimalistic design only for brief search. Leave details for later.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,27 @@
...
+
+## How to use?
+* Just type repo name, push Search button and have short list of related best rated repos.
+* Select repo from list and get shot information about key features of it.
+* Go back and repeat for new search.
+
+## Credits
+Thanks for inspiring and collaborating [Granfalloner](https://github.com/Granfalloner). He has a lot things to do, but he keep stay in touch and helping me with advices.
```
Comment (Granfalloner) [inline README.md:19]: 🙂

**Review by Granfalloner — APPROVED** [review]

**Comment by mudritskiy** [comment]
@Granfalloner What can I do here? Just merge or I have to wait for some action from your side?

**Comment by Granfalloner** [comment]
@mudritskiy If PR is approved, you can merge it yourself.

**Pull request merged**